tables/v1/project_configuration_type_mapping.sql

```diff
@@ -0,0 +1,19 @@
+SET SEARCH_PATH TO v1;
+
+CREATE TABLE IF NOT EXISTS project_configuration_type_mapping (
+    configuration_type         configuration_type        NOT NULL,
+    project_fact_type_enum_id  INTEGER                   NOT NULL,
+    created_at                 TIMESTAMP WITH TIME ZONE  NOT NULL  DEFAULT CURRENT_TIMESTAMP,
+    created_by                 TEXT                      NOT NULL,
+    PRIMARY KEY (configuration_type, project_fact_type_enum_id)
+);
```

Comment:
If we want to support multiple configuration types out of the gate, then I would use a project to configuration type mapping instead. We happen to configure our internal Imbi instance with a "Configuration System" fact type but that is not part of Imbi ... it is our configuration. Maybe there should be a nullable column in `v1.projects` for the configuration backend instead of a separate table 💭

Reply:
I understand we happen to have it that way, but I thought it was a good way to prevent duplication. I guess this gets at the issue of what's the difference between project information and project facts. There's better data integrity and less Imbi configuration needed if we tack the configuration system type directly on the `v1.projects` table, but then it can get duplicated in facts. Perhaps that's still a better option... for now...